### PR TITLE
Release 5.2.5

### PR DESCRIPTION
**Pull request by fredericDelaporte**

### CODE REVIEW

File: releasenotes.txt [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+Release notes - NHibernate - Version 5.2.5
+
+5 issues were resolved in this release.
+
+** Bug
+
+   * #2075 Missing cast when comparing a guid and string columns in SAP SQL Anywhere
+   * #2046 Fix nullable Guid ToString is not translated correctly on some dialects
+   * #2043 System.Reflection.TargetException when an interface is used as class mapping proxy definition
+   * #2040 Incorrect SQL for cast inside a linq join (MS SQL)
```
Comment (hazzik) [inline releasenotes.txt]: I've changed the title for this issue. Please update. Otherwise looks good.

**Comment by hazzik** [comment]
@fredericDelaporte do you really want to wait another month for this?

**Comment by fredericDelaporte** [comment]
No. It is not either that I am waiting the due date I was believing to have set for this version for the 31/03. It is just I have other things to do in my life, and I am not the only one who can publish a release.
I will likely do it this week-end, if not done by someone else before.

**Comment by hazzik** [comment]
I'm asking only because the date on the milestone was set to 30th of April, which is a whole month.

**Pull request merged**